DEV-Simple.xcodeproj/project.pbxproj

```diff
@@ -590,11 +558,8 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = R9SWHSQNV8;
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(inherited)",
-					"$(PROJECT_DIR)/Carthage/Build/iOS",
-				);
+				DEVELOPMENT_TEAM = 6RSSBKDAGB;
```

Comment:
It's great that you've tested this locally. I'm just dropping a small reminder for when you make the PR "ready for review" to revert this change so we can maintain the original development team, thanks! 🙂

Reply:
I've reverted the development team change and added a `NSPhotoLibraryUsageDescription` to the info.plist 🙂